### PR TITLE
update default value of tidb_analyze_version in v5.1 and v5.2

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
update default value of tidb_analyze_version in v5.1 and v5.2

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/8364 https://github.com/pingcap/tidb/issues/31748

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by yiwen92** [comment]
LGTM

**Review by yiwen92 — COMMENTED** [review]
looks nice

File: statistics.md [hunk 1]
```diff
@@ -4,7 +4,14 @@ title: 统计信息简介
...
 
-TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1、v5.2、v5.3 中，该变量的默认值为 `2`，作为实验特性启用。
+TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。`tidb_analyze_version` 在各版本的默认值说明如下：
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本，v5.2.4 及之后的 v5.2.x 版本  |   `1`  |
+|   v5.3.x 及后续版本     |   `2`    |
```
Comment (qiancai) [inline statistics.md:14]: 对于 v5.3.x  及后续版本，也是 “`2`，作为实验特性使用”？

File: statistics.md [hunk 2]
```diff
@@ -4,7 +4,14 @@ title: 统计信息简介
...
 # 统计信息简介
 
-TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1、v5.2、v5.3 中，该变量的默认值为 `2`，作为实验特性启用。
+TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。`tidb_analyze_version` 在各版本的默认值说明如下：
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本，v5.2.4 及之后的 v5.2.x 版本  |   `1`  |
```
Comment (qiancai) [inline statistics.md:13]: v5.2.4 还没发？暂时可以先不提？

Comment (TomShawn) [reply statistics.md:13]:
```suggestion
|   v5.1.4 及之后的 v5.1.x 版本  |   `1`  |
```

File: system-variables.md [hunk 3]
```diff
@@ -279,7 +279,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 这个变量用于控制 TiDB 收集统计信息的行为。
-- 在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1.0 中，该变量的默认值为 `2`，作为实验特性使用，具体可参照[统计信息简介](/statistics.md)文档。
+- 该变量的默认值说明如下，具体可参照[统计信息简介](/statistics.md)文档。
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本，v5.2.4 及之后的 v5.2.x 版本  |   `1`  |
+|   v5.3.x 及后续版本     |   `2`    |
```
Comment (TomShawn) [inline system-variables.md:289]:
```suggestion
|   v5.3.x 及后续版本     |   `2`，作为实验特性使用    |
```

File: system-variables.md [hunk 4]
```diff
@@ -279,7 +279,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 默认值：`2`
 - 这个变量用于控制 TiDB 收集统计信息的行为。
-- 在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1.0 中，该变量的默认值为 `2`，作为实验特性使用，具体可参照[统计信息简介](/statistics.md)文档。
+- 该变量的默认值说明如下，具体可参照[统计信息简介](/statistics.md)文档。
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本，v5.2.4 及之后的 v5.2.x 版本  |   `1`  |
```
Comment (TomShawn) [inline system-variables.md:288]:
```suggestion
|   v5.1.4 及之后的 v5.1.x 版本  |   `1`  |
```

File: statistics.md [hunk 5]
```diff
@@ -4,7 +4,14 @@ title: 统计信息简介
...
 
-TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1、v5.2、v5.3 中，该变量的默认值为 `2`，作为实验特性启用。
+TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。`tidb_analyze_version` 在各版本的默认值说明如下：
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本  |   `1`  |
+|   v5.3.x 及后续版本     |   `2`，作为实验特性使用  |
```
Comment (TomShawn) [inline statistics.md:14]:
```suggestion
```

File: system-variables.md [hunk 6]
```diff
@@ -279,7 +279,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 这个变量用于控制 TiDB 收集统计信息的行为。
-- 在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1.0 中，该变量的默认值为 `2`，作为实验特性使用，具体可参照[统计信息简介](/statistics.md)文档。
+- 该变量的默认值说明如下，具体可参照[统计信息简介](/statistics.md)文档。
+
+|  版本  |   默认值   |
+|  :-       |      :-         |
+|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
+|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性使用      |
+|   v5.1.4 及之后的 v5.1.x 版本  |   `1`  |
+|   v5.3.x 及后续版本     |   `2`，作为实验特性使用    |
```
Comment (TomShawn) [inline system-variables.md:289]:
```suggestion
```

File: system-variables.md [hunk 7]
```diff
@@ -279,7 +279,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 可选值：`1` 和 `2`
 - 默认值：`2`
 - 这个变量用于控制 TiDB 收集统计信息的行为。
-- 在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1.0 中，该变量的默认值为 `2`，作为实验特性使用，具体可参照[统计信息简介](/statistics.md)文档。
+- 该变量的默认值说明如下，具体可参照[统计信息简介](/statistics.md)文档。
```
Comment (TomShawn) [inline system-variables.md:282]:
```suggestion
- `tidb_analyze_version` 在 TiDB v5.1.x 各版本的默认值说明如下，具体可参照[统计信息简介](/statistics.md)文档。
```

File: system-variables.md (hunk 6 above)
Comment (TomShawn) [inline system-variables.md:289]:
```suggestion
|  v5.1.0 - v5.1.3  |  `2`，作为实验特性使用      |
|  v5.1.4 及之后的 v5.1.x 版本  |   `1`  |
```

**Review by winoros — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
LGTM

**Review by yiwen92 — COMMENTED** [review]
LGTM

**Pull request merged**